digdag-server/src/main/java/io/digdag/server/rs/AuthenticatedResource.java

```diff
@@ -18,12 +18,12 @@ protected int getSiteId()
     {
         // siteId is set by JwtAuthInterceptor
         // TODO validate before causing NPE. Improve guice-rs to call @PostConstruct
-        return getAuthenticatedUser().getSiteId();
+        return (int) request.getAttribute("siteId");
     }
 
     protected Config getUserInfo()
     {
-        return getAuthenticatedUser().getUserInfo();
+        return (Config) request.getAttribute("userInfo");
```

Comment:
`master` did not use` getAuthenticatedUser`, but instead fetched the value from `HttpServletRequest`.